internal/validations/validations.go

```diff
@@ -76,13 +83,17 @@ func validateGoSymbols(_ context.Context, path string, baton *Baton) *types.Vali
 		return nil
 	}
 
+	requiredGolangSymbols := requiredGolangSymbolsPre122
```

Comment:
the conditions should have been reversed, its breaking[1] checks for golang < 1.22 .

[1] https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openstack-k8s-operators_ovn-operator/309/pull-ci-openstack-k8s-operators-ovn-operator-main-ovn-operator-build-deploy-kuttl/1803310581785038848/artifacts/ovn-operator-build-deploy-kuttl/openstack-k8s-operators-fips-check/build-log.txt

Reply:
Pushed https://github.com/openshift/check-payload/pull/197